auth/manager.go

```diff
@@ -85,6 +85,7 @@ func (m Manager) Authorize(caller Caller, a Action) error {
 	// Check each caller role against configured role ACLs
 	allowed := false
 	opName := ""
+roles:
```

Comment:
Generally speaking labels make the code harder to follow and would stay way if possible. You can usually restructure the code in other ways without resorting to labels.

Reply:
Updated.